### PR TITLE
Add ooniprobe v2.0.0 install instructions, cleanup, fix URLs

### PR DESCRIPTION
**Pull request by anadahz**
- Add installation instructions for ooniprobe v2.0.0
- Update git repository URLs
- Cleanup obsolete old documentation

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -44,54 +44,6 @@ Furthermore, ooniprobe takes no precautions to protect the install target machin
...
-If you're using the Debian package, you will be asked when installing whether
-you would like to run OONI daily. On other platforms, if you would like to
-contribute measurements to OONI daily you can also add this to your crontab::
-
-    @daily ooniprobe $THE_OONI_COMMAND
-
-Run this command to automatically update your crontab::
-
-    (crontab -l 2>/dev/null; echo "@daily ooniprobe $THE_OONI_COMMAND") | crontab -
-
```
Comment (hellais) [inline README.rst]: Why was this removed? I still think it's useful to have a very concise summary of how to setup ooniprobe on most systems. I guess now it's no longer needed to do the oonideckgen stuff, but still the installation instructions for getting ooniprobe from backports and pip apply.

File: README.rst [hunk 2]
```diff
@@ -116,17 +68,24 @@ Make sure you have installed the following dependencies:
   * libpcap-dev
   * libssl-dev
   * libffi-dev
+  * libgmp-dev
+  * openssl
+  * libyaml-dev
```
Comment (hellais) [inline README.rst]: This is also not required, since the pyyaml will work also without the yaml dev headers.

File: README.rst [hunk 3]
```diff
@@ -116,17 +68,24 @@ Make sure you have installed the following dependencies:
   * libpcap-dev
   * libssl-dev
   * libffi-dev
+  * libgmp-dev
+  * openssl
```
Comment (hellais) [inline README.rst]: This should be libssl-dev.

File: README.rst [hunk 4]
```diff
@@ -179,134 +138,41 @@ Using ooniprobe
...
-    ooniprobe manipulation/http_header_field_manipulation
-
-
-It is also possible to specify inputs to tests as URLs::
-
-
-    ooniprobe blocking/http_requests -f httpo://ihiderha53f36lsd.onion/input/37e60e13536f6afe47a830bfb6b371b5cf65da66d7ad65137344679b24fdccd1
-
-
-You can find the result of the test in your current working directory.
```
Comment (anadahz) [inline README.rst]: Yes it makes sense for this part to be placed in docs.

File: README.rst (hunk 3 above)
Comment (anadahz) [inline README.rst]: `libssl-dev` is already in dependencies: https://github.com/TheTorProject/ooni-probe/pull/649#discussion-diff-84693609R69

Comment (hellais) [inline README.rst]: In that case I believe this dependency is not needed.

File: README.rst (hunk 1 above)
Comment (anadahz) [inline README.rst]: Do we still need the crontabs?
The pip way to install ooniprobe is still in place, install subsection: `Unix systems (with pip)`.

Comment (hellais) [inline README.rst]: No the cronjobs are not needed, but it's just nice to have a section that explains ooniprobe quicky in some simple steps without having to dig into the rest of the docs to find the relevant section.

**Comment by anadahz** [comment]
I think it makes sense to add apt (Debian and Ubuntu package manager) specific install instructions when the ooniprobe package is in stable or in backports.

**Comment by anadahz** [comment]
Updated README with ooniprobe install instructions by using the torproject's package repository.

**Comment by hellais** [comment]
I am going to proceed in merging this
